service/frontend/api/handler.go

```diff
@@ -3312,10 +3312,11 @@ func constructRestartWorkflowRequest(w *types.WorkflowExecutionStartedEventAttri
 		TaskStartToCloseTimeoutSeconds:      w.TaskStartToCloseTimeoutSeconds,
 		Identity:                            identity,
 		WorkflowIDReusePolicy:               types.WorkflowIDReusePolicyTerminateIfRunning.Ptr(),
+		CronOverlapPolicy:                   w.CronOverlapPolicy,
 	}
 	startRequest.CronSchedule = w.CronSchedule
 	startRequest.RetryPolicy = w.RetryPolicy
-	startRequest.DelayStartSeconds = w.FirstDecisionTaskBackoffSeconds
```

Comment:
would not this cause contention if user does bulk restart on bunch of workflows that were previously jittered using this field?